docs/tutorial/anonlink-novt.ipynb

```diff
@@ -491,9 +512,16 @@
     "!anonlink hash 'data/ncvr_numrec_5000_modrec_2_ocp_20_myp_2_nump_10.csv' secret 'novt_schema.json' 'novt_clk_2.json'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`anonlink` provides a command `describe` to inspect the hashing results i.e. the Cryptographic Longterm Key (CLK). Normally we will expect a relative symmetric shape popcount with a moderate mean comparing to bloom filter length."
```

Comment:
`anonlink` provides **the** command `describe`... . It will show the distribution of popcounts as well as a short summary.